pkg/universe.dagger.io/go/container.cue

```diff
@@ -18,25 +18,31 @@ import (
 	// Use go image
 	_image: #Image
 
-	_sourcePath: "/src"
-	_cachePath:  "/root/.cache/gocache"
+	_sourcePath:     "/src"
+	_modCachePath:   "/root/.cache/go-mod"
+	_buildCachePath: "/root/.cache/go-build"
 
 	docker.#Run & {
 		input:   *_image.output | docker.#Image
-		workdir: "/src"
-		command: name: "go"
+		workdir: _sourcePath
```

Comment:
Why removed `go` prefix? It was useful to simplify UX

Reply:
Because for `golangci-lint`, I want to use `go.#Container` (to get the same cache mounts) but I don't want to run `go` (I want to run `golangci-lint`)